elasticsearch/examples/kubesphere/Makefile

```diff
@@ -3,12 +3,13 @@ default: test
 include ../../../helpers/examples.mk
 
 PREFIX := elasticsearch
+RELEASE := elasticsearch-data
```

Comment:
why adding `RELEASE ` ？It isn't used anywhere.

Reply:
In actual, make test  need this params by `include ../../../helpers/examples.mk`. However, `make test` has some problem in test. Shell we remove this function?

Reply:
We can remove it for now, setting to `elasticsearch-data` while ignoring master node doesn't look good?

Reply:
Ok. In this case, the `test` function take a http request to check whether the service of the elasticsearch-data nodes is ok or not.

Reply:
I see, you can comment this line instead of removing it

Reply:
OK